templates/taxi/driver_list.html

```diff
@@ -1,30 +1,35 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
-    </h1>
+  <h1>Driver List
+  </h1>
 
-    {% if driver_list %}
+  {% if driver_list %}
     <table class="table">
       <tr>
         <th>ID</th>
         <th>Username</th>
         <th>First name</th>
         <th>Last name</th>
         <th>License number</th>
+        <th>Update</th>
+        <th>Delete</th>
       </tr>
-    {% for driver in driver_list %}
-      <tr>
-        <td>{{ driver.id }}</td>
-        <td><a href="{{ driver.get_absolute_url }}">{{ driver.username }} {% if user == driver %} (Me){% endif %}</a></td>
-        <td>{{ driver.first_name }}</td>
-        <td>{{ driver.last_name }}</td>
-        <td>{{ driver.license_number }}</td>
-      </tr>
-    {% endfor %}
+      {% for driver in driver_list %}
+        <tr>
+          <td>{{ driver.id }}</td>
+          <td><a href="{{ driver.get_absolute_url }}">{{ driver.username }} {% if user == driver %} (Me){% endif %}</a>
+          </td>
+          <td>{{ driver.first_name }}</td>
+          <td>{{ driver.last_name }}</td>
+          <td>{{ driver.license_number }}</td>
+          <td><a href="{% url 'taxi:driver-update' pk=driver.id %}">UPDATE</a></td>
+          <td><a href="{% url 'taxi:driver-delete' pk=driver.id %}">DELETE</a></td>
```

Comment:
Code Style Issue: The URL identification parameter `name` in the `{% url %}` template tag should use `-` to split words, not `_`. Please, replace 'taxi:driver-delete' with 'taxi:driver-delete'.